File: Components/BlogML/Xml/BlogMLCategory.vb
Imports System.Xml.Serialization

Namespace BlogML.Xml
 <Serializable>
 Public NotInheritable Class BlogMLCategory
  Inherits BlogMLNode

  <XmlAttribute("description")>
  Public Property Description As String

  <XmlAttribute("parentref")>
  Public Property ParentRef As String

 End Class
End Namespace

File: Server/Core/BlogML/Xml/BlogMLCategory.cs
using System;
using System.Xml.Serialization;

namespace DotNetNuke.Modules.Blog.BlogML.Xml
{
  [Serializable]
  public sealed class BlogMLCategory : BlogMLNode
  {

    [XmlAttribute("description")]
    public string Description { get; set; }

    [XmlAttribute("parentref")]
    public string ParentRef { get; set; }

  }
}